### DISCUSSION TITLE
房子早就过剩了,为啥现在还有期房?

### DISCUSSION THREAD

**Discussion by xcb425** [WeNeedHome/SummaryOfLoanSuspension#139] [Ideas]
我认为,中小企业,没有资金先修建好房子,没有调研部门研究到底该不该建房的,就不配去修房子.
大企业你能修,你就修,修不起就滚,让有资格的来修.哪有买东西先全款然后还贷30年,到货还要等卖家大发慈悲给东西的事情?
另外金融创新,也别几把创新了,都是诈骗.金融监管到底有没有用?装修都知道没收工30%定金,就这样还有很多拿了定金就跑的,为啥要给他们钱啊?
银行想快速放贷,地产商想快速拿钱.这两家联合起来操作业主,怎么玩的赢?

**Comment by xcb425** [comment]
另外还有精装修房子的,也是纯纯的智商税

**Reply by Lanweifang** [reply]
可能财政需要卖地收入

**Comment by Besame521** [comment]
老爷们想赚钱啊
